### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by anatolii-zarytskyi**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/cars/addDriver.jsp [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
```
Comment (SerKisJD) [inline src/main/webapp/WEB-INF/views/cars/addDriver.jsp:1]: youy don't use taglib here

File: src/main/webapp/WEB-INF/views/cars/addDriver.jsp [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add driver to car</title>
+</head>
+<body>
+<h1>Enter driver id and car id</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
```
Comment (SerKisJD) [inline src/main/webapp/WEB-INF/views/cars/addDriver.jsp:9]: make paragraphs for better reading, below same

File: src/main/java/mate/controller/car/CreateCarController.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/add")
+public class CreateCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+    private static final ManufacturerService manufacturerService = (ManufacturerService) injector
```
Comment (SerKisJD) [inline src/main/java/mate/controller/car/CreateCarController.java:20]: private final

**Review by SerKisJD — CHANGES_REQUESTED** [review]
review all jsp files for paragraphs, and all services in your controllers should be private final

**Review by SerKisJD — APPROVED** [review]

**Review by Resci — APPROVED** [review]

**Pull request closed without merging**